src/common/src/catalog/column.rs

```diff
@@ -211,6 +211,38 @@ impl From<&ColumnDesc> for ProstColumnDesc {
     }
 }
 
+impl From<&DataType> for ColumnDesc {
```

Comment:
I suggest removing this "general" `From` implementation as it uses dummy values for `name`, `column_id`, and other fields, which is unsuitable for other cases.

Reply:
use 2 functions instead.